### ISSUE TITLE
Add support for QML language

### ISSUE THREAD

**Issue by nocnokneo** [joeblau/gitignore.io#277]
Ignore *.qmlc and *.js files (cached binary versions of their .qml counterpart, similar to .pyc and .py Python files)

```
### QML ###
# Cached binary representations of QML and JS files
*.qmlc
*.jsc
```

**Comment by joeblau** [comment]
Can you submit a PR and add this template in `data/custom`?

**Comment by joeblau** [comment]
Fixed by #278
